[ZT] Update Gateway global rules (#8398)
cloudflare/cloudflare-docs: content/cloudflare-one/policies/filtering/global-rules.md

---
pcx_content_type: reference
title: Global rules
layout: single
weight: 6
---

# Global rules

Cloudflare Zero Trust applies a set of global rules to all accounts.

> ## DNS rules

Reviewer: What do you think about calling these sections DNS/Network/HTTP policies to match our policy builder?

Author: I'm okay with that. In that case, should we change the page name to "Global policies" to match our wording elsewhere?

| Criteria | Value                                                                                                                                                                                             | Action | Description                                            |
| -------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------ | ------------------------------------------------------ |

> | Hostname | `*.nel.cloudflare.com`                                                                                                                                                                            | allow  | Allows L4 access to SNI domains for WARP registration. |

Reviewer: Should this be `*.net.cloudflare.com`?

Author: `nel` is correct according to the source of truth.

Author: `nel` is correct according to the source of truth.

> | Hostname | `*.cloudflareclient.com`                                                                                                                                                                          | allow  | Allows L4 access for clients.                          |

Reviewer: ```suggestion
| Hostname | `*.cloudflareclient.com`                                                                                                                                                                          | allow  | Allows L4 access for the WARP client.                          |
```

| Hostname | `*.cloudflare-gateway.com`                                                                                                                                                                        | allow  | Allows L4 access for the Gateway proxy with PAC files. |

> | Hostname | `dash.teams.cloudflare.com`, `help.teams.cloudflare.com`, `blocked.teams.cloudflare.com`, `api.cloudflare.com`, `cloudflarestatus.com`, `www.cloudflarestatus.com`, and `one.dash.cloudflare.com` | allow  | Allows L4 access for Zero Trust applications.          |

Reviewer: By Zero Trust applications, do you mean general Zero Trust features? To me, Zero Trust applications are the same thing as Cloudflare Access applications. Is there a reason these particular domains are grouped together?

| Hostname | `*.cloudflareaccess.com`                                                                                                                                                                          | allow  | Allows L4 access for Cloudflare Access applications.   |

## Proxy rules

| Criteria         | Value                       | Action | Description                                                                                                                                    |
| ---------------- | --------------------------- | ------ | ---------------------------------------------------------------------------------------------------------------------------------------------- |

> | Hostname         | `*.cloudflareaccess.com`    | allow  | Allows L7 security for Cloudflare Access applications.                                                                                         |

Reviewer: Not sure if "Allows L4 access" and "Allows L7 security" are clear to users.

| Hostname       | `api.cloudflare.com`                                         | bypass    | Bypasses Cloudflare's API endpoint.                                                                             |
| Hostname       | `dash.teams.cloudflare.com`                                  | bypass    | Prevents users from being locked out of the Zero Trust dashboard.                                               |
| Hostname       | `*.dash.cloudflare.com`                                      | bypass    | Bypasses the Cloudflare dashboard and subdomains.                                                               |
| Hostname       | `blocked.teams.cloudflare.com`                               | bypass    | Prevents an infinite loop on the Gateway block page.                                                            |
| Hostname       | `developers.cloudflare.com` and `help.cloudflarebrowser.com` | noisolate | Prevents isolation of Cloudflare developer docs and help pages to help users troubleshoot configuration issues. |

> | Hostname       | `*.assets.browser.run`                                       | bypass    | Required for [Remote Browser Isolation (RBI)](/cloudflare-one/policies/browser-isolation/) access.              |

Reviewer: ```suggestion
| Hostname       | `*.assets.browser.run`                                       | bypass    | Required for [Remote Browser Isolation (RBI)](/cloudflare-one/policies/browser-isolation/).             |
```

| Hostname       | `*.edge.browser.run` and `*.cloudflarebrowser.com`           | bypass    | Required for RBI access.                                                                                        |

> | Hostname       | `*.edge.browser.run` and `*.cloudflarebrowser.com`           | isolate   | Required for RBI access.                                                                                        |

Reviewer: ```suggestion
| Hostname       | `*.edge.browser.run` and `*.cloudflarebrowser.com`           | isolate   | Required for RBI.                                                                                        |
```
